packages/create-nx-workspace/src/create-empty-workspace.ts

```diff
@@ -32,15 +32,16 @@ export async function createEmptyWorkspace<T extends CreateWorkspaceOptions>(
     options.packageManager = packageManager;
   }
 
-  const directory = getFileName(name);
+  options.name = getFileName(name);
+  const directory = options.name;
 
   const args = unparse({
     ...options,
   }).join(' ');
 
   const pmc = getPackageManagerCommand(packageManager);
 
-  const command = `new ${directory} ${args}`;
+  const command = `new ${args}`;
```

Comment:
The `${directory}` here was useless. The `name` arg takes precedence over the positional arg and overwrites it.

I'm normalizing the `name` option above, which is a known and required option, so we don't rely on the `new` generator rederiving it. We could also pass the `directory` option directly, but that option is not in the `new` generator schema for some reason.